packages/asciinema.rb

```diff
@@ -3,24 +3,12 @@
 class Asciinema < Package
   description 'Terminal session recorder'
   homepage 'https://asciinema.org/'
-  version '2.0.2-2'
+  @_ver = '2.1.0'
+  version @_ver
   license 'GPL-3+'
   compatibility 'all'
-  source_url 'https://github.com/asciinema/asciinema/archive/v2.0.2.tar.gz'
-  source_sha256 '2578a1b5611e5375771ef6582a6533ef8d40cdbed1ba1c87786fd23af625ab68'
-
-  binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/asciinema/2.0.2-1_armv7l/asciinema-2.0.2-1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/asciinema/2.0.2-1_armv7l/asciinema-2.0.2-1-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/asciinema/2.0.2-2_i686/asciinema-2.0.2-2-chromeos-i686.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/asciinema/2.0.2-2_x86_64/asciinema-2.0.2-2-chromeos-x86_64.tpxz'
-  })
-  binary_sha256({
-    aarch64: '5fddd233f9f093935c23574dc1ce6247103352e845408375fa537fd2f56675ec',
-     armv7l: '5fddd233f9f093935c23574dc1ce6247103352e845408375fa537fd2f56675ec',
-       i686: '9086a6bce66d3ad0542d8d61f6904bd24078998d2a0867ced9b65eabb15b889b',
-     x86_64: 'a94b73de95817df65ede7ba0e58c71547cf7eb6a326b9707dd94a29a3fd73aab'
-  })
```

Comment:
No binaries?  Building from git is not normal user-friendly.

Reply:
This took less than 1 min to build...

Reply:
binaries added